src/tools/singlejar/zip64_test.sh

```diff
@@ -53,7 +53,15 @@ else
 fi
 
 singlejar="$(rlocation "io_bazel/src/tools/singlejar/singlejar${EXE_EXT}")"
-jartool="$(rlocation "local_jdk/bin/jar${EXE_EXT}")"
+javabase="$1"
+if [[ $javabase = /* ]]; then
+  jartool="$1/bin/jar${EXE_EXT}"
+else
+  if [[ $javabase = external/* ]]; then
```

Comment:
What if javabase is in the main repo?
Does it start with the main repo's name then?

Reply:
@laszlocsomor yes